.eslintignore

```diff
@@ -1 +1 @@
-**/test/**/expected.js
```

Comment:
actually i have included `actual.js` (or how those files are named at the moment) on purpose, i know we sometimes might want to write code that breaks our lint rules, but we can always use eslint-ignore comments for that

 WDYT?

Reply:
imho `fixtures` could be excluded completely
1 we don't care about styles there, those are for machine, not for people
2 I'd prefer don't touch them if we decide to change code style once

Reply:
I don't have a strong opinion on this, but I don't believe that "those are for machine" is a right statement here.

They are our tests and they need be reviewable by humans for them to have any purpose.